okhttp/src/main/kotlin/okhttp3/internal/cache/DiskLruCache.kt

```diff
@@ -225,6 +228,8 @@ class DiskLruCache internal constructor(
       }
     }
 
+    civilizedFileSystem = fileSystem.isCivilized(journalFileBackup)
```

Comment:
This is probably worthy of logging when false.

Reply:
ooooh I think it’d be pretty boring. It’ll pretty much always be true on Windows, and false elsewhere.